media-video/jellyfin-media-player/metadata.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "https://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+	<maintainer type="person">
+		<email>arsen@aarsen.me</email>
+		<name>Arsen Arsenović</name>
+	</maintainer>
+	<maintainer type="project">
```

Comment:
I'm the maintainer for this package in ::guru, would you mind adding me here too?

```
<email>telans@posteo.de</email>
<name>James Beddek</name>
```